app/src/main/java/org/pesmypetcare/mypetcare/services/community/CommunityAdapter.java

```diff
@@ -406,7 +412,17 @@ public void unlikePost(User user, Post post) {
 
     @Override
     public void reportPost(User user, Post post, String reportMessage) {
-        // Not implemented yet
+        ExecutorService executorService = Executors.newSingleThreadExecutor();
+        executorService.execute(() -> {
+            try {
+                ServiceLocator.getInstance().getForumManagerClient().reportMessage(user.getToken(),
+                        post.getForum().getGroup().getName(), post.getForum().getName(), post.getUsername(), user.getUsername(),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 128).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5643405)